parl/framework/policy_distribution.py

```diff
@@ -70,22 +70,31 @@ def entropy(self):
 
         return entropy
 
-    def logp(self, actions):
+    def logp(self, actions, safe_eps=1e-6):
         """
         Args:
             actions: An int64 tensor with shape [BATCH_SIZE]
+            safe_eps: A scalar of safe epsilon to avoid returning extremely large value
```

Comment:
Is`eps`  a more commonly used name ?

eps: a small float constant that avoids overflows

Reply:
done; eps: a small float constant that avoids underflows